src/main/resources/Dockerfile

```diff
@@ -31,5 +31,5 @@ ENV TARANTOOL_INSTANCES_FILE=$TARANTOOL_INSTANCES_FILE
 ENV TARANTOOL_CLUSTER_COOKIE=$TARANTOOL_CLUSTER_COOKIE
 COPY $CARTRIDGE_SRC_DIR $TARANTOOL_WORKDIR
 WORKDIR $TARANTOOL_WORKDIR
-CMD cartridge build && cartridge start --run-dir=$TARANTOOL_RUNDIR --data-dir=$TARANTOOL_DATADIR \
+CMD cartridge build --verbose && cartridge start --run-dir=$TARANTOOL_RUNDIR --data-dir=$TARANTOOL_DATADIR \
```

Comment:
Looks like this flag needs to be optional (as an env variable), otherwise synchronous logging produces too much output and is too slow, as you may see in the CI results

Reply:
It's PR for debug, we were curious why after https://github.com/tarantool/testcontainers-java-tarantool/pull/83 CI fall all the time